src/coreComponents/linearAlgebra/interfaces/MatrixBase.hpp

```diff
@@ -617,6 +619,27 @@ class MatrixBase : public virtual LinearOperator< VECTOR >
     R.multiply( AP, dst );
   }
 
+  /**
+   * @brief Compute the triple product <tt>dst = P1^T * this * P2</tt>
+   * @param P1 the left "prolongation" matrix
+   * @param P2 the right "prolongation" matrix
+   * @param dst the resulting product matrix (will be re-created as needed)
+   */
+  virtual void multiplyPtAP( Matrix const & P1,
```

Comment:
Should a different name be used? Like this it suggest that a there is a unique prolongation operator.

Reply:
`multiplyP1tAP2` is unambiguous